guide/popular-topics/intents.md

```diff
@@ -1,97 +1,68 @@
 
 ## Enabling Intents
 
-You can choose which intents you'd like to receive as client options when instantiating your bot client.
+As touched on before you can now choose which events you want to receive from Discord. In order to do so you need to pass the respective intents for the events you want to receive to the Client constructor.
 
 A list of all available gateway intents the library supports can be found at [the discord.js documentation](https://discord.js.org/#/docs/main/stable/class/Intents?scrollTo=s-FLAGS). The events included in the respective intents on the [discord API documentation](https://discord.com/developers/docs/topics/gateway#list-of-intents).
 
-:::tip
-`GUILD_PRESENCES` is required in order to receive the initial GuildMember data when your bot connects to Discord. If you do not supply it, your member caches will start empty. `guildMemberUpdate` events will not be processed, regardless of if the `GUILD_MEMBER` partial is enabled, unless the `GuildMember` has been cached by other means such as by sending a message, being mentioned in one, or the `guildMemberAdd` event. Before you disable intents think about what your bot does and how not receiving the listed events might prevent it from doing this. Version 12 of discord.js may not function as expected when specific intents are not provided.
+:::warning
+Note that discord.js relies heavily on caching to provide its functionality. Some methods that seem unrelated might stop working if certain events do not arrive.
+
+Please make sure to provide the full list of gateway intents and partials you use in your Client constructor when asking for support on our [Discord server](https://discord.gg/bRCvFy9) or [GitHub repository](https://github.com/discordjs/discord.js).
 :::
 
 ```js
 const { Client } = require('discord.js');
 const client = new Client({ ws: { intents: ['GUILDS', 'GUILD_MESSAGES'] } });
 ```
 
-## The Intents bit field wrapper
-
-Discord.js provides a utility structure [`Intents`](https://discord.js.org/#/docs/main/stable/class/Intents) which can be utilized to easily adapt the underlying bit field.
+## The Intents Bitfield
 
-We also provide static fields for all, privileged and non-privileged intents. You can provide these as-is or pass them to the Intents constructor to further modify to your needs.
+Discord.js provides a utility structure [`Intents`](https://discord.js.org/#/docs/main/stable/class/Intents) which can be utilized to easily adapt the underlying bit field. This class also has static fields for all (`Intents.ALL`), privileged (`Intents.PRIVILEGED`) and non-privileged (`Intents.NON_PRIVILEGED`) intents. You can provide these as-is or pass them to the Intents constructor to further modify to your needs.
 
 ```js
 const { Client, Intents } = require('discord.js');
-const client = new Client({ ws: { intents: Intents.ALL } });
+const client = new Client({ ws: { intents: Intents.NON_PRIVILEGED } });
```

Comment:
I wouldn't include this because it goes against the purpose of intents and should not be used, but if this is here users will probably just copy it

Reply:
Removed the entire section in aad8a10 and clarified that the static attributes are meant to be used as templates rather than as-is.